lively.ide/studio/world-browser.cp.js

```diff
@@ -704,14 +702,15 @@ export class WorldBrowserModel extends ViewModel {
   }
 
   async displayItems () {
-    this.reset();
     const { loadingIndicator } = this.ui;
-    loadingIndicator.position = pt(this.view.width / 2 - loadingIndicator.width / 2, this.view.height / 2 - loadingIndicator.height / 2);
-    loadingIndicator.animate({
-      opacity: 1, duration: 300
-    });
+    if (!loadingIndicator.visible) {
+      await loadingIndicator.animate({
+        visible: true, duration: 300
+      });
+    }
     let entities = this.playgroundsMode ? await this.db.latestCommits('world') : await Project.listAvailableProjects();
     // Filter out the project that is opened anyways.
+    this.reset();
```

Comment:
```suggestion
    this.reset();
    // Filter out the project that is opened anyways.
```

Just to keep the comment attached to its line